test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py

```diff
@@ -40,7 +40,7 @@ def default_autoware_launch_package_of(architecture_type):
             f"architecture_type := {architecture_type.perform(context)} is not supported. Choose one of {architecture_types()}."
         )
     return {
-        "awf/auto": "scenario_test_runner_launch",
```

Comment:
Please let me know where is the scenario_test_runner package is?

Reply:
It is [here](https://gitlab.com/robotecai-public/AutowareAuto/-/merge_requests/9), to be merged with `scenario_simulator_v2_changes` branch